cves/CVE-2015-1831.yml

```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: The issue with this CVE was the input validaion in Apache 2.3.20. Specifically the excludeParams function overrode those patterens in DefaultExcludedPatternsChecker,
```

Comment:
Looks like the yaml parses wrong here. You need a pipe. Look at how the others are done.

Reply:
Also, this reads like the NVD description. Explain it to a mere mortal without talking about versions. Your readers will NOT have taken 331.

Reply:
Added the Pipe and changed the description to be more reader friendly.